### PR TITLE
[SPARK-45715][CORE] QueryPlanningTracker::measurePhase minor refactor

### PR DESCRIPTION
**Pull request by xuzifu666**
### What changes were proposed in this pull request?

 refactor some codes in QueryPlanningTracker::measurePhase
startTime  and endTime keep the same

### Why are the changes needed?

code refactor 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

not need

### CODE REVIEW

**Comment by xuzifu666** [comment]
cc @HeartSaVioR PTAL

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala [hunk 1]
```diff
@@ -134,7 +134,7 @@ class QueryPlanningTracker(
    * and the recorded end time will be the end time of the last call.
    */
   def measurePhase[T](phase: String)(f: => T): T = {
-    val startTime = System.currentTimeMillis()
+    val startTime = System.currentTimeMillis
```
Comment (LuciferYang) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala:137]: Is this a known compilation error that will be triggered in Scala 3? Or is it an existing compilation warning in Scala 2.13? If not, I don't think it's worth fixing it just for the sake of maintaining consistency in call style.

Comment (xuzifu666) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala:137]: no error or warning，just a code refactor for it，starttime with() and endtime not，is a little strange @LuciferYang

Comment (HyukjinKwon) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala:137]: then let's just leave it.

Comment (xuzifu666) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala:137]: ok, close it

**Pull request closed without merging**